build_tools/cirrus/arm_wheel.yml

```diff
@@ -36,3 +36,39 @@ macos_arm64_wheel_task:
 
   wheels_artifacts:
     path: "wheelhouse/*"
+
+linux_arm64_wheel_task:
+  compute_engine_instance:
+    image_project: cirrus-images
+    image: family/docker-builder-arm64
+    architecture: arm64
+    platform: linux
+    cpu: 4
+    memory: 4G
+  env:
+    CONFTEST_PATH: ${CIRRUS_WORKING_DIR}/conftest.py
+    CONFTEST_NAME: conftest.py
+    CIBW_ENVIRONMENT: OMP_NUM_THREADS=2
+                      OPENBLAS_NUM_THREADS=2
+                      SKLEARN_SKIP_NETWORK_TESTS=1
+                      SKLEARN_BUILD_PARALLEL=5
+                      CPU_COUNT=4
+    CIBW_TEST_COMMAND: bash {project}/build_tools/github/test_wheels.sh
```

Comment:
Maybe we should have a `build_tools/common` folder for scripts that are used by more than one CI.

Reply:
I would have the folder named: `build_tools/wheels` to place all the scripts for wheel building.